transmart-server/src/main/groovy/org/transmart/server/subsctiption/SubscriptionFrequency.groovy

```diff
@@ -0,0 +1,29 @@
+package org.transmart.server.subsctiption
+
+import groovy.transform.CompileStatic
+
+/**
+ * The frequency of emails for user query subscription
+ */
+@CompileStatic
+enum SubscriptionFrequency {
+
+    DAILY ('DAILY'),
+    WEEKLY ('WEEKLY')
+
+    private String subscriptionFrequency
```

Comment:
Enum names match exactly with their `subscriptionFrequency` values. I don't see a point in having this field so far.